src/Settings/components/translation.jsx

```diff
@@ -0,0 +1,88 @@
+import { ListItem, Avatar, Switch } from "@rneui/themed";
+import colors from "../../common/colors";
+import STRINGS from "../../common/localization";
+import {
+  toggleEnglishTranslation,
+  togglePunjabiTranslation,
+  toggleSpanishTranslation,
+} from "../../common/actions";
+import { useSelector } from "react-redux";
+import { useState } from "react";
+
+function TranslationComponent({ isNightMode, dispatch }) {
+  const { isEnglishTranslation, isSpanishTranslation, isPunjabiTranslation } = useSelector(
+    (state) => state
+  );
+  const [isExpanded, toggleIsExpanded] = useState(false);
```

Comment:
instead of toggleIsExpanded, you should use "setIsExpanded" as state setter function.

Reply:
same reason as above. I can rename this from "toggleIsExpended" to "toggleExpended"

Reply:
@amitojsingh veerji, let's analyze the functionality of the function. It sets a new value in the state.

```js
toggleIsExpanded(!isExpanded);
```

In the given code, the !isExpanded part enables the toggle functionality. The function toggleIsExpanded simply sets the provided value. So naming that `setIsExpanded` would make the name more bound to the functionality. 

I might be knit-picking veerji, just trying to make our code follow the best practices. I'm open to discussing this further.